<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlansWindow</class>
 <widget class="QDialog" name="PlansWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="backToMenuButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(170, 170, 127);
gridline-color: rgb(108, 108, 108);
background-color: rgb(230, 230, 230);
selection-background-color: rgb(205, 225, 255);</string>
   </property>
   <property name="text">
    <string>Back to menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>70</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 500 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>PLANS FOR TODAY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(170, 170, 127);
gridline-color: rgb(108, 108, 108);
background-color: rgb(230, 230, 230);
selection-background-color: rgb(205, 225, 255);</string>
   </property>
   <property name="text">
    <string>Delete task</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Plan a new task</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>690</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(170, 170, 127);
gridline-color: rgb(108, 108, 108);
background-color: rgb(230, 230, 230);
selection-background-color: rgb(205, 225, 255);</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="planLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 235, 235)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(170, 170, 127);
gridline-color: rgb(108, 108, 108);
background-color: rgb(230, 230, 230);
selection-background-color: rgb(205, 225, 255);</string>
   </property>
   <property name="text">
    <string>Add task</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>120</y>
     <width>411</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="plansLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="horizontalSpacing">
     <number>7</number>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="archivButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(170, 170, 127);
gridline-color: rgb(108, 108, 108);
background-color: rgb(230, 230, 230);
selection-background-color: rgb(205, 225, 255);</string>
   </property>
   <property name="text">
    <string>Archive task</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>531</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="archivLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="archivDeleteButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(170, 170, 127);
gridline-color: rgb(108, 108, 108);
background-color: rgb(230, 230, 230);
selection-background-color: rgb(205, 225, 255);</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="archivLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>600</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 235, 235)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ARCHIVE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
